Packs/CiscoSMA/Integrations/CiscoSMA/CiscoSMA.py

```diff
@@ -1851,7 +1851,7 @@ def main() -> None:
     filter_value = params.get("filter_value")
     recipient_filter_operator = params.get("recipient_filter_operator")
     recipient_filter_value = params.get("recipient_filter_value")
-
+    timeout = arg_to_number(params.get("timeout")) if params.get("timeout") else None
```

Comment:
```suggestion
    timeout = arg_to_number(params.get("timeout"))
```

arg_to_number returns None when None is passed, which means we do not need to check if it is None (Also the default value of get it None)